s3/src/main/scala/akka/stream/alpakka/s3/impl/HttpRequests.scala

```diff
@@ -11,12 +11,35 @@ import akka.http.scaladsl.model.{ContentTypes, RequestEntity, _}
 import akka.stream.alpakka.s3.S3Settings
 import akka.stream.scaladsl.Source
 import akka.util.ByteString
-
 import scala.collection.immutable.Seq
 import scala.concurrent.{ExecutionContext, Future}
 
 private[alpakka] object HttpRequests {
 
+  def listBucket(
+      bucket: String,
+      region: String,
+      prefix: Option[String] = None,
+      continuation_token: Option[String] = None
+  )(implicit conf: S3Settings): HttpRequest = {
+
+    val listTypeQuery: (String, String) = "list-type" -> "2"
+    val prefixQuery: Option[(String, String)] = prefix.map("prefix" -> _)
+    val continuationTokenQuery: Option[(String, String)] =
+      continuation_token.map("continuation-token" -> _.replaceAll("=", "%3D"))
```

Comment:
Why does only this character need to be replaced? Are you use that's correct at all, because `Uri.query` will also encode characters, so that the percent character may then be double encoded?